acceptance/upgrade_test.go

```diff
@@ -0,0 +1,85 @@
+package acceptance
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+)
+
+func Test_HydraUpgrade(t *testing.T) {
+	type Case struct {
+		Name        string
+		BeforeImage string
+		AfterImage  string
+	}
+
+	cases := []Case{
+		{
+			Name:        "upgrade from columnar ext rename",
+			BeforeImage: "ghcr.io/hydrasdb/hydra:7149160_8eee2a8_dacf51c",
```

Comment:
Maybe this could be in a variable (ENV maybe?), maybe it could represent what's currently in production, and we update it somehow? Defer to @ljcesca on ideas here.

Reply:
Yeah, I can move it to an ENV. It prob. would be nice to have a public endpoint that returns the version that we are using in prod.

Reply:
Extracted the prod version to https://github.com/HydrasDB/hydra/pull/31/commits/3757663f1fb7b09d3c099b4f1d223f30a482bd83